specification/dimensions/commitmentdiscounttype.md

```diff
@@ -0,0 +1,39 @@
+# Commitment Discount Type
```

Comment:
Do we need to clarify how this is not the same as a negotiated discount you get when you commit to a spend-based commitment to a provider - as a part of your contract negotiation? 'Contractual commitment' below could be ambiguous.

Reply:
Yes, this makes sense. What is the term we used for the account-level, negotiated discounts?

Reply:
@udam-f2, I'm assuming you're referring to the CBD description in the first paragraph. Since we're sourcing that from the CommitmentDiscountId PR, can you review the description there and leave a comment if you feel it's not covering anything in particular? I'll resolve this for this PR and we can update as needed.